arangod/Cluster/ClusterFeature.h

```diff
@@ -66,7 +66,11 @@ class ClusterFeature : public application_features::ApplicationFeature {
   std::string _myRole;
   std::string _myEndpoint;
   std::string _myAdvertisedEndpoint;
-  uint32_t _systemReplicationFactor = 2;
+  std::uint32_t _defaultReplicationFactor = 0; // a value of 0 means it will use the min replication factor 
+  std::uint32_t _systemReplicationFactor = 2;
+  std::uint32_t _minReplicationFactor = 1;     // minimum replication factor (0 = unrestricted)
```

Comment:
```suggestion
  std::uint32_t _minReplicationFactor = 1; 
```

1 is already unrestricted, you cannot create any collection lower than this.